std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
>I'm getting weary of this style, are we promoting this? I mean Importing each range primitive in every scope, if anything I'd expect it to be top level and full.

As we discussed before, I'd agree that it should be pulled out to outer scopes if used a lot, though not module scope, but right now I see only five imports of `popFront` from `std.range.primitives` in this 3+ kloc file, with dozens of uses of some other `popFront`.  In these cases, I think it's worthwhile to explicitly and locally document which `popFront` is being used.

I don't think this style is being promoted for all D code, certainly not without automation, but I think it makes the standard library more readable.